scripts/ci/benchmarks-ci.sh

```diff
@@ -21,6 +21,8 @@ pallets=(
 	frame_system
 )
 
+mkdir -p $benhcmarkOutput
```

Comment:
```suggestion
mkdir -p $benhcmarkOutput || echo "Directory $benhcmarkOutput exists"
```

Otherwise it'll fail if directory exists

Reply:
No, it shouldn't because `-p` option takes care about it.